applications/machine_learning/README.rst

```diff
@@ -159,6 +182,7 @@ The application supports the following build types:
 * ``ZDebug`` -- Debug version of the application - can be used to verify if the application works correctly.
 * ``ZRelease`` -- Release version of the application - can be used to achieve better performance and reduce memory consumption.
 
+Not every board supports both mentioned build types.
```

Comment:
btw, why there is no Release version for nrf5340dk?

Reply:
I discussed this with Paweł and we decided to support only Debug variant here to limit number of configurations that we need to maintain.